eyereturn/DOCUMENTATION.md

```diff
@@ -0,0 +1,100 @@
+# Eyereturn
+## General Compatibility
+|Feature|  |
+|---|---|
+| Consent | no |
+| Native Ad Support | no |
+| SafeFrame Support | no |
+| PMP Support | yes |
+ 
+## Browser Compatibility
+| Browser |  |
+|--- |---|
+| Chrome | yes |
+| Edge | yes |
+| Firefox | yes |
+| Internet Explorer 9 | yes (adapter debugger doesn't work in ie9) |
+| Internet Explorer 10 | yes |
+| Internet Explorer 11 | yes |
+| Safari | yes |
+| Mobile Chrome | yes |
+| Mobile Safari | yes |
+| UC Browser | no |
+| Samsung Internet | no |
+| Opera | yes |
+ 
+## Adapter Information
+| Info | |
+|---|---|
+| Partner Id | EyereturnHtb |
+| Ad Server Responds in (Cents, Dollars, etc) | Dollars |
+| Bid Type (Gross / Net) | Net |
+| GAM Key (Open Market) | |
+| GAM Key (Private Market) | |
+| Ad Server URLs | https://prometheus-ix.eyereturn.com/prometheus/bid, https://p3.eyereturn.com/|
+| Slot Mapping Style (Size / Multiple Sizes / Slot) | Slot |
```

Comment:
Based on the __generateRequestObj function it seems the request requires sizes. Please update to the documentation with the correct `Slot Mapping Style` type as per `Ad Slot-Level Configuration` section in the KB: https://kb.indexexchange.com/Adapters/Adapter_Debugger.htm